### PR TITLE
Refactored and cleaned-up some PHP code

### PR DESCRIPTION
**Pull request by DanReyLop**
Sorry for the ambiguous PR title, these were just a few things that I encountered and I couldn't leave as they were. They are separated in independent commits for easier reviewing.

@allendav @nabsul @jeffstieler

### CODE REVIEW

**Comment by jeffstieler** [comment]
@DanReyLop it looks like this broke the tests. Can you look into it?

**Comment by DanReyLop** [comment]
No way of running PHPUnit on Windows :cry: 
This can take a while.

**Comment by DanReyLop** [comment]
I've just removed the tests that were failing. Before, the scripts and styles were being included conditionally, but that's not really needed because "registering" a script/style doesn't actually include it in the page. So I've removed that logic, and the test that covered that logic.

File: woocommerce-connect-client.php [hunk 1]
```diff
@@ -419,48 +419,9 @@ public function woocommerce_payment_gateways( $payment_gateways ) {
...
-				// If we are on a system status page, make sure it is the connect tab
-				if ( 'connect' !== $tab ) {
-					return;
-				}
-			} else {
-				// Don't recognize the hook? Go no further
-				return;
-			}
-
+		public function admin_enqueue_scripts() {
```
Comment (allendav) [inline woocommerce-connect-client.php]: Won't this cause our big ol' bundles to be loaded on every single wp-admin page?

File: client/views/shipping/style.scss [hunk 2]
```diff
@@ -1,12 +0,0 @@
-.wcc-form-success {
-    animation: disappear .3s ease-in-out 2s;
```
Comment (allendav) [inline client/views/shipping/style.scss]: Why this change?

Comment (DanReyLop) [inline client/views/shipping/style.scss]: This file was not used, dead code removal :)

File: woocommerce-connect-client.php (hunk 1 above)
Comment (DanReyLop) [inline woocommerce-connect-client.php]: `wp_register_script( 'handler_name', 'path/to/script' )` just *registers* that for that name (`'handler_name'`) those files need to be included. The bundle (and the CSS, and the translations) won't actually be included unles `wp_enqueue_script( 'handler_name' )` and `wp_enqueue_style( 'handler_name' )` are called.

So this code is pretty harmless on its own, the extra logic to not register the script and styles when not needed was not worth it, in my opinion.

You can try it if you don't believe me 100%, the result is obvious with the Chrome DevTools open :)

**Comment by jeffstieler** [comment]
Looks good and tests well (doesn't appear to break the self help page). :shipit:
